app/views/dogs/index.html.erb

```diff
@@ -20,15 +20,14 @@
 <%= will_paginate @dogs %>
 
   <% if signed_in? && session[:mgr_view] == true %>
-
+    
     <%= render 'manager' %>
-
+    
```

Comment:
@blai00 what are you using for an editor/IDE?

Reply:
brackets for MAC, should I be using something else? I am not particularly fond of it so any suggestion would be nice

Reply:
Try out https://code.visualstudio.com/